Parsely/wp-parsely
# Add Recommendations Block

## Description
Introduce a `Recommendations Block` that displays links to relevant pages, powered by Parse.ly's Recommendations (`/related`) API.

## Motivation and Context
See: https://github.com/Parsely/web/issues/12585

## How to perform manual testing
1. Add the following line to an mu-plugin to enable the flags for the API Endpoint and Block:
    ```
    add_filter( 'wp_parsely_recommendations_block_enabled', '__return_true' );
    ```
2. Specify a valid API key on the plugin's options page, in order to test against a "real" site's recommendations.
3. Drop the block in various places (a post, page, full-site editor).
4. Tinker with different Block settings to see the effects they have.

## Screenshots
![Screenshot from 2022-02-02 17-28-02](https://user-images.githubusercontent.com/1587282/152248311-23c41952-9dcb-4610-97d8-560e48267782.png)
